_integration-schemas/selligent/program.md

```diff
@@ -0,0 +1,47 @@
+---
+tap: "selligent"
+version: 1.0
+key: ""
+
+name: "program"
+doc-link: 
+singer-schema: https://github.com/singer-io/tap-selligent/blob/master/tap_selligent/schemas.py
+description: "This table contains program data."
+
+
+replication-method: "Full Table"
+
+replication-key:
```

Comment:
We can remove this because it's not being used, as the table uses Full Table rep.